language/de/acp/ban.php

```diff
@@ -77,11 +77,11 @@
 	'PERMANENT'		=> 'Dauerhaft',
 
 	'UNTIL'						=> 'Bis zum',
-	'USER_BAN'					=> 'Einen oder mehrere Benutzernamen sperren',
+	'USER_BAN'					=> 'Einen oder mehrere Benutzer mittels Benutzernamen sperren',
```

Comment:
Es wird in ACP > Tab: Benutzer und Gruppen > Benutzernamen sperren als Kopf vom Fieldset genutzt "legend" für ein Fieldset und als Überschrift über dem erklärendem Absatz genutzt. 

Von daher wäre wohl: "Benutzer anhand des Benuternamens sperren" besser, da kürzer.